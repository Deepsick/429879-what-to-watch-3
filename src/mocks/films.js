export const movies = [
  {
    name: `Fantastic Beasts: The Crimes of Grindelwald`,
    picture: `fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  },
  {
    name: `Bohemian Rhapsody`,
    picture: `bohemian-rhapsody.jpg`,
  },
  {
    name: `Macbeth`,
    picture: `macbeth.jpg`,
  },
  {
    name: `Aviator`,
    picture: `aviator.jpg`,
  },
  {
    name: `We need to talk about Kevin`,
    picture: `we-need-to-talk-about-kevin.jpg`,
  },
  {
    name: `What We Do in the Shadows`,
    picture: `what-we-do-in-the-shadows.jpg`,
  },
  {
    name: `Revenant`,
    picture: `revenant.jpg`,
  },
  {
    name: `Johnny English`,
    picture: `johnny-english.jpg`,
  },
];

export const movie = {
  name: `The Grand Budapest Hotel`,
  genre: `drama`,
  year: 2014,
};
