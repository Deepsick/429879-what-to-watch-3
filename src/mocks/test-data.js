export const movie = {
  name: `The Grand Budapest Hotel`,
  genre: `drama`,
  year: 2014,
};

export const onMovieTitleClick = () => {};
export const onCardHover = () => {};
export const onMouseOut = () => {};
export const mockFunction = () => {};
export const hoverDelay = 1000;
export const isShown = true;
export const moviesCount = 8;
export const muted = true;
export const isPlaying = true;
export const genre = `drama`;
export const trailer = `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`;
export const isVideo = false;
export const mockBool = true;
export const TabName = {
  OVERVIEW: `overview`,
  DETAILS: `details`,
  REVIEWS: `reviews`,
};

export const movies = [
  {
    name: `Fantastic Beasts: The Crimes of Grindelwald`,
    picture: `fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    id: `1`,
    trailer: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
  },
  {
    name: `Bohemian Rhapsody`,
    picture: `bohemian-rhapsody.jpg`,
    id: `2`,
    trailer: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
  },
  {
    name: `Macbeth`,
    picture: `macbeth.jpg`,
    id: `3`,
    trailer: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
  },
  {
    name: `Aviator`,
    picture: `aviator.jpg`,
    id: `4`,
    trailer: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
  },
  {
    name: `We need to talk about Kevin`,
    picture: `we-need-to-talk-about-kevin.jpg`,
    id: `5`,
    trailer: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  },
  {
    name: `What We Do in the Shadows`,
    picture: `what-we-do-in-the-shadows.jpg`,
    id: `6`,
    trailer: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  },
  {
    name: `Revenant`,
    picture: `revenant.jpg`,
    id: `7`,
    trailer: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  },
  {
    name: `Johnny English`,
    picture: `johnny-english.jpg`,
    id: `8`,
    trailer: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
  },
];

export const detailedMovies = [
  {
    name: `Fantastic Beasts: The Crimes of Grindelwald`,
    poster: `fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    genre: `drama`,
    id: `1`,
    trailer: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    year: 2014,
    cover: `bg-the-grand-budapest-hotel.jpg`,
    director: `Wes Andreson`,
    starring: [`Bill Murray`, `Edward Norton`, `Jude Law`, `Willem Dafoe`],
    description: `
      In the 1930s, the Grand Budapest Hotel is a popular European ski resort,
      presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, 
      becomes Gustave's friend and protege.
  
      Gustave prides himself on providing first-class service to the hotel's guests,
      including satisfying the sexual needs of the many elderly women who stay there.
      When one of Gustave's lovers dies mysteriously, Gustave finds himself the recipient of a
      priceless painting and the chief suspect in her murder.
    `,
  },
  {
    name: `Bohemian Rhapsody`,
    poster: `bohemian-rhapsody.jpg`,
    genre: `drama`,
    id: `2`,
    trailer: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    year: 2014,
    cover: `bg-the-grand-budapest-hotel.jpg`,
    director: `Wes Andreson`,
    starring: [`Bill Murray`, `Edward Norton`, `Jude Law`, `Willem Dafoe`],
    description: `
      In the 1930s, the Grand Budapest Hotel is a popular European ski resort,
      presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, 
      becomes Gustave's friend and protege.
  
      Gustave prides himself on providing first-class service to the hotel's guests,
      including satisfying the sexual needs of the many elderly women who stay there.
      When one of Gustave's lovers dies mysteriously, Gustave finds himself the recipient of a
      priceless painting and the chief suspect in her murder.
    `,
  },
  {
    name: `Macbeth`,
    poster: `macbeth.jpg`,
    genre: `drama`,
    id: `3`,
    trailer: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    year: 2014,
    cover: `bg-the-grand-budapest-hotel.jpg`,
    director: `Wes Andreson`,
    starring: [`Bill Murray`, `Edward Norton`, `Jude Law`, `Willem Dafoe`],
    description: `
      In the 1930s, the Grand Budapest Hotel is a popular European ski resort,
      presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, 
      becomes Gustave's friend and protege.
  
      Gustave prides himself on providing first-class service to the hotel's guests,
      including satisfying the sexual needs of the many elderly women who stay there.
      When one of Gustave's lovers dies mysteriously, Gustave finds himself the recipient of a
      priceless painting and the chief suspect in her murder.
    `,
  },
  {
    name: `Aviator`,
    poster: `aviator.jpg`,
    genre: `drama`,
    id: `4`,
    trailer: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    year: 2014,
    cover: `bg-the-grand-budapest-hotel.jpg`,
    director: `Wes Andreson`,
    starring: [`Bill Murray`, `Edward Norton`, `Jude Law`, `Willem Dafoe`],
    description: `
      In the 1930s, the Grand Budapest Hotel is a popular European ski resort,
      presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, 
      becomes Gustave's friend and protege.
  
      Gustave prides himself on providing first-class service to the hotel's guests,
      including satisfying the sexual needs of the many elderly women who stay there.
      When one of Gustave's lovers dies mysteriously, Gustave finds himself the recipient of a
      priceless painting and the chief suspect in her murder.
    `,
  },
  {
    name: `We need to talk about Kevin`,
    poster: `we-need-to-talk-about-kevin.jpg`,
    genre: `drama`,
    id: `5`,
    trailer: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
    year: 2014,
    cover: `bg-the-grand-budapest-hotel.jpg`,
    director: `Wes Andreson`,
    starring: [`Bill Murray`, `Edward Norton`, `Jude Law`, `Willem Dafoe`],
    description: `
      In the 1930s, the Grand Budapest Hotel is a popular European ski resort,
      presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, 
      becomes Gustave's friend and protege.
  
      Gustave prides himself on providing first-class service to the hotel's guests,
      including satisfying the sexual needs of the many elderly women who stay there.
      When one of Gustave's lovers dies mysteriously, Gustave finds himself the recipient of a
      priceless painting and the chief suspect in her murder.
    `,
  },
  {
    name: `What We Do in the Shadows`,
    poster: `what-we-do-in-the-shadows.jpg`,
    genre: `drama`,
    id: `6`,
    trailer: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
    year: 2014,
    cover: `bg-the-grand-budapest-hotel.jpg`,
    director: `Wes Andreson`,
    starring: [`Bill Murray`, `Edward Norton`, `Jude Law`, `Willem Dafoe`],
    description: `
      In the 1930s, the Grand Budapest Hotel is a popular European ski resort,
      presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, 
      becomes Gustave's friend and protege.
  
      Gustave prides himself on providing first-class service to the hotel's guests,
      including satisfying the sexual needs of the many elderly women who stay there.
      When one of Gustave's lovers dies mysteriously, Gustave finds himself the recipient of a
      priceless painting and the chief suspect in her murder.
    `,
  },
  {
    name: `Revenant`,
    poster: `revenant.jpg`,
    genre: `drama`,
    id: `7`,
    trailer: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
    year: 2014,
    cover: `bg-the-grand-budapest-hotel.jpg`,
    director: `Wes Andreson`,
    starring: [`Bill Murray`, `Edward Norton`, `Jude Law`, `Willem Dafoe`],
    description: `
      In the 1930s, the Grand Budapest Hotel is a popular European ski resort,
      presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, 
      becomes Gustave's friend and protege.
  
      Gustave prides himself on providing first-class service to the hotel's guests,
      including satisfying the sexual needs of the many elderly women who stay there.
      When one of Gustave's lovers dies mysteriously, Gustave finds himself the recipient of a
      priceless painting and the chief suspect in her murder.
    `,
  },
  {
    name: `Johnny English`,
    poster: `johnny-english.jpg`,
    genre: `comedy`,
    id: `8`,
    trailer: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
    year: 2014,
    cover: `bg-the-grand-budapest-hotel.jpg`,
    director: `Wes Andreson`,
    starring: [`Bill Murray`, `Edward Norton`, `Jude Law`, `Willem Dafoe`],
    description: `
      In the 1930s, the Grand Budapest Hotel is a popular European ski resort,
      presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, 
      becomes Gustave's friend and protege.
  
      Gustave prides himself on providing first-class service to the hotel's guests,
      including satisfying the sexual needs of the many elderly women who stay there.
      When one of Gustave's lovers dies mysteriously, Gustave finds himself the recipient of a
      priceless painting and the chief suspect in her murder.
    `,
  },
];


