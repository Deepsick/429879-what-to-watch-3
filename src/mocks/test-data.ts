export const mockFunction = () => {};
export const hoverDelay = 1000;
export const isShown = true;
export const moviesCount = 8;
export const muted = true;
export const isPlaying = true;
export const duration = 1000;
export const rating = 12345;
export const genre = `drama`;
export const trailer = `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`;
export const isVideo = false;
export const id = 1;
export const mockString = ``;
export const Auth = {
  NO_AUTH: `NO_AUTH`,
  AUTH: `AUTH`,
};
export const user = {
  name: 'user',
  avatar: 'hello.jpg',
}
export const review = {
  rating: 8,
  comment: 'hello',
};
export const error = 'Невозможно получить данные';
export const comments = [
  {
    id: 1,
    comment: '123213',
    user: {
      id: 3,
      name: 'asdw',
    },
    rating: 111,
    date: '123',
  },
  {
    id: 2,
    comment: 'hi',
    user: {
      id: 5,
      name: 'asdwssss',
    },
    rating: 6664,
    date: '123',
  },
]

export const mockBool = true;
export const TabName = {
  OVERVIEW: `overview`,
  DETAILS: `details`,
  REVIEWS: `reviews`,
};

export const movies = [
  {
    bgColor: '#ffffff',
    preview: ``,
    video: ``,
    rating: 3,
    scoresCount: 300,
    duration: 100500,
    isFavorite: true,
    name: `Fantastic Beasts: The Crimes of Grindelwald`,
    poster: `fantastic-beasts-the-crimes-of-grindelwald.jpg`,
    genre: `drama`,
    id: 1,
    trailer: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    year: 2014,
    cover: `bg-the-grand-budapest-hotel.jpg`,
    director: `Wes Andreson`,
    starring: [`Bill Murray`, `Edward Norton`, `Jude Law`, `Willem Dafoe`],
    description: `
      In the 1930s, the Grand Budapest Hotel is a popular European ski resort,
      presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, 
      becomes Gustave's friend and protege.
  
      Gustave prides himself on providing first-class service to the hotel's guests,
      including satisfying the sexual needs of the many elderly women who stay there.
      When one of Gustave's lovers dies mysteriously, Gustave finds himself the recipient of a
      priceless painting and the chief suspect in her murder.
    `,
  },
  {
    bgColor: '#ffffff',
    preview: ``,
    video: ``,
    rating: 3,
    scoresCount: 300,
    duration: 100500,
    isFavorite: true,
    name: `Bohemian Rhapsody`,
    poster: `bohemian-rhapsody.jpg`,
    genre: `drama`,
    id: 2,
    trailer: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    year: 2014,
    cover: `bg-the-grand-budapest-hotel.jpg`,
    director: `Wes Andreson`,
    starring: [`Bill Murray`, `Edward Norton`, `Jude Law`, `Willem Dafoe`],
    description: `
      In the 1930s, the Grand Budapest Hotel is a popular European ski resort,
      presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, 
      becomes Gustave's friend and protege.
  
      Gustave prides himself on providing first-class service to the hotel's guests,
      including satisfying the sexual needs of the many elderly women who stay there.
      When one of Gustave's lovers dies mysteriously, Gustave finds himself the recipient of a
      priceless painting and the chief suspect in her murder.
    `,
  },
  {
    bgColor: '#ffffff',
    preview: ``,
    video: ``,
    rating: 3,
    scoresCount: 300,
    duration: 100500,
    isFavorite: true,
    name: `Macbeth`,
    poster: `macbeth.jpg`,
    genre: `drama`,
    id: 3,
    trailer: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    year: 2014,
    cover: `bg-the-grand-budapest-hotel.jpg`,
    director: `Wes Andreson`,
    starring: [`Bill Murray`, `Edward Norton`, `Jude Law`, `Willem Dafoe`],
    description: `
      In the 1930s, the Grand Budapest Hotel is a popular European ski resort,
      presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, 
      becomes Gustave's friend and protege.
  
      Gustave prides himself on providing first-class service to the hotel's guests,
      including satisfying the sexual needs of the many elderly women who stay there.
      When one of Gustave's lovers dies mysteriously, Gustave finds himself the recipient of a
      priceless painting and the chief suspect in her murder.
    `,
  },
  {
    bgColor: '#ffffff',
    preview: ``,
    video: ``,
    rating: 3,
    scoresCount: 300,
    duration: 100500,
    isFavorite: true,
    name: `Aviator`,
    poster: `aviator.jpg`,
    genre: `drama`,
    id: 4,
    trailer: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`,
    year: 2014,
    cover: `bg-the-grand-budapest-hotel.jpg`,
    director: `Wes Andreson`,
    starring: [`Bill Murray`, `Edward Norton`, `Jude Law`, `Willem Dafoe`],
    description: `
      In the 1930s, the Grand Budapest Hotel is a popular European ski resort,
      presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, 
      becomes Gustave's friend and protege.
  
      Gustave prides himself on providing first-class service to the hotel's guests,
      including satisfying the sexual needs of the many elderly women who stay there.
      When one of Gustave's lovers dies mysteriously, Gustave finds himself the recipient of a
      priceless painting and the chief suspect in her murder.
    `,
  },
  {
    bgColor: '#ffffff',
    preview: ``,
    video: ``,
    rating: 3,
    scoresCount: 300,
    duration: 100500,
    isFavorite: true,
    name: `We need to talk about Kevin`,
    poster: `we-need-to-talk-about-kevin.jpg`,
    genre: `drama`,
    id: 5,
    trailer: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
    year: 2014,
    cover: `bg-the-grand-budapest-hotel.jpg`,
    director: `Wes Andreson`,
    starring: [`Bill Murray`, `Edward Norton`, `Jude Law`, `Willem Dafoe`],
    description: `
      In the 1930s, the Grand Budapest Hotel is a popular European ski resort,
      presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, 
      becomes Gustave's friend and protege.
  
      Gustave prides himself on providing first-class service to the hotel's guests,
      including satisfying the sexual needs of the many elderly women who stay there.
      When one of Gustave's lovers dies mysteriously, Gustave finds himself the recipient of a
      priceless painting and the chief suspect in her murder.
    `,
  },
  {
    bgColor: '#ffffff',
    preview: ``,
    video: ``,
    rating: 3,
    scoresCount: 300,
    duration: 100500,
    isFavorite: true,
    name: `What We Do in the Shadows`,
    poster: `what-we-do-in-the-shadows.jpg`,
    genre: `drama`,
    id: 6,
    trailer: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
    year: 2014,
    cover: `bg-the-grand-budapest-hotel.jpg`,
    director: `Wes Andreson`,
    starring: [`Bill Murray`, `Edward Norton`, `Jude Law`, `Willem Dafoe`],
    description: `
      In the 1930s, the Grand Budapest Hotel is a popular European ski resort,
      presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, 
      becomes Gustave's friend and protege.
  
      Gustave prides himself on providing first-class service to the hotel's guests,
      including satisfying the sexual needs of the many elderly women who stay there.
      When one of Gustave's lovers dies mysteriously, Gustave finds himself the recipient of a
      priceless painting and the chief suspect in her murder.
    `,
  },
  {
    bgColor: '#ffffff',
    preview: ``,
    video: ``,
    rating: 3,
    scoresCount: 300,
    duration: 100500,
    isFavorite: true,
    name: `Revenant`,
    poster: `revenant.jpg`,
    genre: `drama`,
    id: 7,
    trailer: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
    year: 2014,
    cover: `bg-the-grand-budapest-hotel.jpg`,
    director: `Wes Andreson`,
    starring: [`Bill Murray`, `Edward Norton`, `Jude Law`, `Willem Dafoe`],
    description: `
      In the 1930s, the Grand Budapest Hotel is a popular European ski resort,
      presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, 
      becomes Gustave's friend and protege.
  
      Gustave prides himself on providing first-class service to the hotel's guests,
      including satisfying the sexual needs of the many elderly women who stay there.
      When one of Gustave's lovers dies mysteriously, Gustave finds himself the recipient of a
      priceless painting and the chief suspect in her murder.
    `,
  },
  {
    bgColor: '#ffffff',
    preview: ``,
    video: ``,
    rating: 3,
    scoresCount: 300,
    duration: 100500,
    isFavorite: true,
    name: `Johnny English`,
    poster: `johnny-english.jpg`,
    genre: `comedy`,
    id: 8,
    trailer: `https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`,
    year: 2014,
    cover: `bg-the-grand-budapest-hotel.jpg`,
    director: `Wes Andreson`,
    starring: [`Bill Murray`, `Edward Norton`, `Jude Law`, `Willem Dafoe`],
    description: `
      In the 1930s, the Grand Budapest Hotel is a popular European ski resort,
      presided over by concierge Gustave H. (Ralph Fiennes). Zero, a junior lobby boy, 
      becomes Gustave's friend and protege.
  
      Gustave prides himself on providing first-class service to the hotel's guests,
      including satisfying the sexual needs of the many elderly women who stay there.
      When one of Gustave's lovers dies mysteriously, Gustave finds himself the recipient of a
      priceless painting and the chief suspect in her murder.
    `,
  },
];


