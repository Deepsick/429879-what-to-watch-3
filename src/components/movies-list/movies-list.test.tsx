import * as React from 'react';
import {Router} from 'react-router-dom';
import * as renderer from 'react-test-renderer';

import history from '../../history';
import MoviesList from './movies-list';
import {movies} from '../../mocks/test-data';

it(`Should render MoviesList component correctly`, () => {
  const node = renderer.create(
      <Router history={history} >
        <MoviesList
          movies={movies}
        />
      </Router>
  ).toJSON();

  expect(node).toMatchSnapshot();
});
