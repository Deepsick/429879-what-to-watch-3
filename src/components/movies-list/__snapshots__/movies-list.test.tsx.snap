// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render MoviesList component correctly 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseOut={[Function]}
  >
    <a
      href="/films/1"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Fantastic Beasts: The Crimes of Grindelwald"
          height="175"
          src=""
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/1"
        onClick={[Function]}
      >
        Fantastic Beasts: The Crimes of Grindelwald
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseOut={[Function]}
  >
    <a
      href="/films/2"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Bohemian Rhapsody"
          height="175"
          src=""
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/2"
        onClick={[Function]}
      >
        Bohemian Rhapsody
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseOut={[Function]}
  >
    <a
      href="/films/3"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Macbeth"
          height="175"
          src=""
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/3"
        onClick={[Function]}
      >
        Macbeth
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseOut={[Function]}
  >
    <a
      href="/films/4"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Aviator"
          height="175"
          src=""
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/4"
        onClick={[Function]}
      >
        Aviator
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseOut={[Function]}
  >
    <a
      href="/films/5"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="We need to talk about Kevin"
          height="175"
          src=""
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/5"
        onClick={[Function]}
      >
        We need to talk about Kevin
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseOut={[Function]}
  >
    <a
      href="/films/6"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="What We Do in the Shadows"
          height="175"
          src=""
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/6"
        onClick={[Function]}
      >
        What We Do in the Shadows
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseOut={[Function]}
  >
    <a
      href="/films/7"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Revenant"
          height="175"
          src=""
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/7"
        onClick={[Function]}
      >
        Revenant
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseOut={[Function]}
  >
    <a
      href="/films/8"
      onClick={[Function]}
    >
      <div
        className="small-movie-card__image"
      >
        <img
          alt="Johnny English"
          height="175"
          src=""
          width="280"
        />
      </div>
    </a>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
        href="/films/8"
        onClick={[Function]}
      >
        Johnny English
      </a>
    </h3>
  </article>
</div>
`;
